src/swap/SwapScreen.tsx

```diff
@@ -640,8 +652,10 @@ export function SwapScreen({ route }: Props) {
           {showMaxSwapAmountWarning && (
             <InLineNotification
               severity={Severity.Warning}
-              title={t('swapScreen.maxSwapAmountWarning.title')}
-              description={t('swapScreen.maxSwapAmountWarning.body')}
+              title={t('swapScreen.maxSwapAmountWarning.title', { tokenSymbol: fromToken?.symbol })}
+              description={t('swapScreen.maxSwapAmountWarning.body', {
```

Comment:
should we use new translation keys so that this is backwards compatible?

Reply:
oh right, good catch!